lib/screens/v2/candidate_generator/widgets/train_crowding.ex

```diff
@@ -88,4 +96,15 @@ defmodule Screens.V2.CandidateGenerator.Widgets.TrainCrowding do
     localized_alert.alert.effect in [:suspension, :shuttle] and
       LocalizedAlert.location(localized_alert) in [:boundary_downstream, :boundary_upstream]
   end
+
+  defp log_crowding_info(prediction, crowding_config) do
+    Enum.each(
+      prediction.vehicle.carriages,
+      fn %Screens.Vehicles.Carriage{} = carriage ->
+        Logger.info(
+          "[train_crowding car_crowding_info] station_id=#{crowding_config.station_id} direction_id=#{crowding_config.direction_id} car_number=#{carriage.car_number} vehicle_id=#{prediction.vehicle.id} crowding_level=#{carriage.occupancy_status} trip_id=#{prediction.trip.id} prediction_id=#{prediction.id}"
+        )
+      end
+    )
+  end
```

Comment:
Instead of separate logging for each car, can we log the array of all car crowding for a vehicle?

Reply:
The main reason for splitting out the cars is for easier grouping and comparisons in Splunk tables/dashboards. I worry that keeping all info in the same log will require some complicated log parsing. I guess we could go with one log and, as long as we ensure sorting is consistent, look for any time that the serialized arrays are not equal. Is that what you had in mind?

Reply:
Yeah, that's what I had in mind. What is the use case for grouping and comparisons on the car level?